apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ExceptionLoggerTest.cs

```diff
@@ -42,12 +43,19 @@ public ExceptionLoggerTest()
         }
 
         [Fact]
-        public async Task ErrorReportingTest()
+        public async Task ErrorReportingTest_ErrorReporting()
+            => await ErrorReportingTest(TestServer.Create<ReportToErrorReportingApplication>());
+
+        [Fact]
+        public async Task ErrorReportingTest_Logging()
+            => await ErrorReportingTest(TestServer.Create<ReportToLoggingApplication>());
```

Comment:
It's not that important since these are tests, but I would remove `async\await` on these two.